### ISSUE TITLE
Add support for Sublime Text 4 "inactive sheet dimming" if possible

### ISSUE THREAD

**Issue by keithrussell42** [Monokai/monokai-pro-sublime-text#142]
Sublime Text 4 is nearing official release (Build 4106 currently). The default themes have a very handy feature. When you have multiple tabs selected the active one has its background lighter than the inactive one which is slightly "dimmed". The new options which seem to be relevant are:
```
    // Controls the style of file tabs for the Default, Default Dark, and
    // Adaptive themes.
    // Options: "rounded", "square"
    //   The value "angled" provides the tab style from Sublime Text 3, but is
    //   only compatible with the Default and Adaptive themes, and will not
    //   support new features like inactive pane dimming.
    "file_tab_style": "rounded",

    // If inactive sheets should have their background slightly modified to
    // make input focus more obvious.
    "inactive_sheet_dimming": true,
```

I'm not sure if Monokai Pro can do the same thing but it is a surprisingly useful visual cue. Without it I end up hunting for the cursor to figure out which is the active tab (or sheet in the new parlance).

**Comment by cig0** [comment]
I'm pretty much waiting for this feature to be implemented in order to acquire the license!

**Comment by EthanHipps** [comment]
@Monokai Any update on whether this will be added or not?

**Comment by valr** [comment]
@Monokai , thanks a lot for this!

**Comment by EthanHipps** [comment]
@Monokai Awesome, Thanks!

**Comment by kevinsjoberg** [comment]
@Monokai given that this was fixed in `1.1.20`, I think we can close this issue. 🙂
